roles/transmissionvpn/tasks/main.yml

```diff
@@ -0,0 +1,83 @@
+#########################################################################
+# Title:            Cloudbox Community: transmissionvpn                     #
+# Author(s):        Tarek                                               #
+# URL:              https://github.com/Cloudbox/Community               #
+# Docker Image(s):  haugene/transmission-openvpn                        #
+# --                                                                    #
+#         Part of the Cloudbox project: https://cloudbox.works          #
+#########################################################################
+#                   GNU General Public License v3.0                     #
+#########################################################################
+---
+- name: "Setting CloudFlare DNS Record"
+  include_role:
+    name: cloudflare-dns
+  vars:
+    subdomain: transmission
+  when: cloudflare_enabled
+
+- name: Stop and remove any existing container
+  docker_container:
+    name: transmission
+    state: absent
+
+- name: Create transmissionvpn directories
+  file: "path={{ item }} state=directory mode=0775 owner={{ user.name }} group={{ user.name }}"
+  with_items:
+    - /opt/transmissionvpn
+
+- name: Create new downloads directories
+  file: "path={{ item }} state=directory mode=0775 owner={{ user.name }} group={{ user.name }} recurse=yes"
+  with_items:
+    - "{{ downloads.torrents }}"
+    - "{{ downloads.torrents }}/transmissionvpn"
+    - "{{ downloads.torrents }}/transmissionvpn/completed"
+    - "{{ downloads.torrents }}/transmissionvpn/incoming"
+    - "{{ downloads.torrents }}/transmissionvpn/watched"
+    - "{{ downloads.torrents }}/transmissionvpn/torrents"
+    - "{{ downloads.torrents }}/transmissionvpn/vpn"
+    - /opt/transmissionvpn/
+  when: (torrents_downloads_path is defined)|default(false)
+
+- name: Set default_volumes variable
+  set_fact:
+    default_volumes:
+      - "/etc/localtime:/etc/localtime:ro"
+      - "/opt/transmissionvpn/vpn:/etc/openvpn/custom"
+      - "/mnt:/mnt"
+      - "/opt/transmissionvpn:/opt/transmissionvpn"
+
+- name: Create and start container
+  docker_container:
+    name: transmissionvpn
+    image: haugene/transmission-openvpn:latest-alpine
+    pull: yes
+    env:
+      PUID: "{{ uid }}"
+      PGID: "{{ gid }}"
+      VIRTUAL_HOST: "transmission.{{ user.domain }}"
```

Comment:
Change virtual host to "transmissionvpn"

Reply:
done